battle-game-functions-jpmaatz/src/script.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+// TODO - write your code here.
+const randomDamage = () => Math.floor(Math.random()*10)+1 
+
+const chooseOption = (opt1,opt2) => Math.random() < 0.5 ? opt1:opt2
+
+function attackPlayer (health){
+    return health - randomDamage;
+}

```

Comment:
We strongly suggest using function expressions rather than declarations:

```
const attackPlayer = function (health) {
    return health - randomDamage
}
```

To follow in the style of your other function expressions, this could also be written with arrow syntax:

```
const attackPlayer = health => health - randomDamage
```